engine-standalone-tracing/src/types/call_tracer.rs

```diff
@@ -0,0 +1,288 @@
+                    context.apparent_value,
+                );
+            }
+            evm::tracing::Event::Create {
+                caller,
+                address,
+                scheme,
+                value,
+                init_code,
+                target_gas,
+            } => {
+                let call_type = match scheme {
+                    evm::CreateScheme::Legacy { .. } => CallType::Create,
+                    evm::CreateScheme::Create2 { .. } => CallType::Create2,
+                    evm::CreateScheme::Fixed(_) => CallType::Create, // is this even possible in production?
+                };
+
+                self.enter(
+                    call_type,
+                    Address::new(caller),
+                    Address::new(address),
+                    init_code.to_vec(),
+                    target_gas.unwrap_or_default(),
+                    value,
+                );
+            }
+            evm::tracing::Event::Suicide {
+                address,
+                target,
+                balance,
+            } => {
+                // TODO: gas = 0 is right?
+                self.enter(
+                    CallType::SelfDestruct,
+                    Address::new(address),
+                    Address::new(target),
+                    Vec::new(),
+                    0,
+                    balance,
+                );
+                self.exit(Vec::new(), None);
+            }
+            // Exit event always comes after RecordRefund, so we don't need to worry about gas here (it's handled in RecordRefund)
+            evm::tracing::Event::Exit {
+                reason,
+                return_value,
+            } => {
+                let error = match reason {
+                    evm::ExitReason::Succeed(_) => None,
+                    other => Some(other),
+                };
+                self.exit(return_value.to_vec(), error);
+            }
+
+            // not useful
+            evm::tracing::Event::TransactCall { .. } => (),
+            evm::tracing::Event::TransactCreate { .. } => (),
+            evm::tracing::Event::TransactCreate2 { .. } => (),
+        }
+    }
+}
+
+#[cfg(feature = "serde")]
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+pub struct SerializableCallFrame {
+    #[serde(rename = "type")]
+    call_type: String,
+    from: String,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    to: Option<String>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    value: Option<String>,
+    gas: String,
+    #[serde(rename = "gasUsed")]
+    gas_used: String,
+    input: String,
+    output: String,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    error: Option<String>,
+    #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    calls: Vec<SerializableCallFrame>,
+}
+
+#[cfg(feature = "serde")]
+impl From<CallFrame> for SerializableCallFrame {
```

Comment:
Could be equivalent to [GetResult](https://github.com/ethereum/go-ethereum/blob/ad15050c7fbedd0f05a49e81400de18c2cc2c284/eth/tracers/native/call.go#L151) if you serialize it as JSON.